frontend/src/metabase/admin/settings/components/EmbeddingSettings/EmbeddingSdkSettings.tsx

```diff
@@ -77,7 +191,48 @@ export function EmbeddingSdkSettings({
             />
           </SetByEnvVarWrapper>
         </Box>
+
+        {isEE && isHosted && (
+          <Box>
+            <SettingHeader
+              id="version-pinning"
+              setting={{
+                display_name: t`Version pinning`,
+                description: t`Metabase Cloud instances are automatically upgraded to new releases. SDK packages are strictly compatible with specific version of Metabase. You can request to pin your Metabase to a major version and upgrade your Metabase and SDK dependency in a coordinated fashion.`,
+              }}
+            />
+            <Button
+              compact
+              variant="outline"
+              leftIcon={<Icon size={12} name="mail" aria-hidden />}
+              component={ExternalLink}
+              fz="0.75rem"
+              href="mailto:help@metabase.com"
+            >{t`Request version pinning`}</Button>
+          </Box>
+        )}
+
+        <Text data-testid="sdk-documentation">
+          {jt`Check out the ${(
+            <ExternalLink key="sdk-doc" href={documentationUrl}>
+              {t`documentation`}
+            </ExternalLink>
+          )} for more.`}
+        </Text>
       </Stack>
     </Box>
   );
 }
+
+function useDocsUrl(url: string) {
```

Comment:
[Links are specced here.](https://www.notion.so/metabase/Feature-flag-and-settings-for-SDK-8ed424c776194528b6332c5fc2510a0c?pvs=4#97a019ce1d0b4d19a50ecfd1a34f060f)